# HYPOTHESIS TESTING

### Question 1

###### Suppose a child psychologist says that the average time that working mothers spend talking to their children is at-least 11 minutes per day. 
###### You want to test
###### You conduct a random sample of 1000 working mothers and find they spend an average of 11.5 minutes per day talking with their children. 
###### Assume prior research suggests the population standard deviation is 2.3 minutes.
###### onduct test with a level of significance of alpha=0.05

### Solution

#### creating null and alternate hypothesis
###### Null hypothesis: The average time that working mothers spend talking to their children is at least 11 minutes per day.
###### Alternate hypothesis: The average time that working mothers spend talking to their children is less than 11 minutes per day.

#### significance level
###### level of significance (alpha)=0.05
###### confidence interval =95%

In [7]:
#### One tailed z test
mu =11
x =11.5
std =2.3
z= (x- mu)/ std
z

0.2173913043478261

#### calculating critical value
###### alpha = 5% =0.05
###### from z table critical value for 0.05 is 1.645

#### Conclusion
###### z value(0.21739) is less than critical value( 1.645) so we accept the hypothesis
###### So, The average time that working mothers spend talking to their children is at least 11 minutes per day.

### Question 2

###### A coffee shop claims that their average wait time for customers is less than 5 minutes.
###### To test this claim, a sample of 40 customers is taken, and their wait times are recorded. 
###### The sample mean wait time is found to be 4.6 minutes with a standard deviation of 0.8 minutes.
###### Perform a hypothesis test at a significance level of 0.05 and determine whether there is enough 
###### evidence to support the coffee shop's claim.
###### Perform the hypothesis testing using correct test statistics and appropriate critical value

### Solution

#### creating null and alternate hypothesis
###### Null hypothesis: The average wait time for customers at the coffee shop is less than 5 minutes.
###### Alternative hypothesis: The average wait time for customers at the coffee shop is 5 minutes or more

#### significance level
###### level of significance (alpha)=0.05
###### confidence interval =95%

In [8]:
#### One tailed z test
mu =5
x =4.6
std =0.8
z= (x- mu)/ std
z

-0.5000000000000004

#### calculating critical value
###### alpha = 5% =0.05
###### from z table critical value for 0.05 is 1.645

#### Conclusion
###### z value(-0.500) is less than critical value( 1.645) so we accept the hypothesis
###### So, The average wait time for customers at the coffee shop is less than 5 minutes.